### ISSUE TITLE
Dynamic SVG path breaks render in Edge

### ISSUE THREAD

**Issue by vdzk** [WebReflection/lighterhtml#65]
When I try to render an SVG path with a dynamic "d" attribute it throws an error in Edge and doesn't render anything. It also throws a similar error in Chrome but renders anyway.
Relevant code:
```javascript
const {render, svg} = lighterhtml
const d = 'M150 0 L75 200 L225 200 Z'
render(document.body, () => svg`
  <svg height="210" width="400">
    <path d=${d} />
  </svg>
`)
```
[Same in a codepen](https://codepen.io/dante_diagram/pen/wvwNYpv).
[Screenshot of the error in Edge.](https://cdn1.imggmi.com/uploads/2019/9/24/77ce3f33914bb87782b3ac36be4ce77a-full.png)
Error in Chrome: <path> attribute d: Expected moveto path command ('M' or 'm'), "-0.223183%".
I would be grateful for a fix.
Thanks!

**Comment by vdzk** [comment]
Yes, Edge renders everything fine if I define the path non dynamically.

**Comment by WebReflection** [comment]
reading the error, it looks like Edge randomly drop the node when value is not valid, so that the rest fail ... I hate that browser 😅

Do you mind trying this instead, and let me know how it goes?

```js
const {render, svg} = lighterhtml
const d = 'M150 0 L75 200 L225 200 Z'
render(document.body, () => svg`
  <svg height="210" width="400">
    <path .d=${d} />
  </svg>
`)
```

Please note the `.d=${...}` instead of `d=${...}`, thanks

**Comment by WebReflection** [comment]
Actually, never mind ... that doesn't seem to work in other browsers ... bummer!

**Comment by WebReflection** [comment]
Closing this, as it is not related to this library, but to part of its core (affecting also hyperHTML, heresy, and others libraries based on _domtagger_).

https://github.com/WebReflection/domtagger/issues/18

**Comment by WebReflection** [comment]
So, this cannot possibly be solved otherwise in _domtagger_, so I am reopening the case (with a fix coming son)

**Comment by WebReflection** [comment]
So here the thing: _IE/Edge_ are an absolute mess with attributes.

The moment these don't recognize the content, they trash it 😱

![Screenshot from 2019-09-24 16-59-32](https://user-images.githubusercontent.com/85749/65531093-0d1d0100-def1-11e9-853a-e39d11c86a23.png)

Accordingly, there's no way _domtagger_ can fix this, because it'll never find the content it's looking for, even if for every other node is just fine.

## The Solution

When it comes to SVGs, if you want to maintain compatibility with IE/Edge, you need to set attributes as _accessor_, meaning that you can use the `.d=${...}` syntax, which will be ignored by the parser, and it will be configured as `node.setAttribute(name, value)` **only** for SVG nodes.

As result, not a single browser would show up any error anymore, but this is a caveat of dealing with SVG and hyper/lighterHTML.

I hope this works for you, and my previously suggested workaround will now work:

```js
const {render, svg} = lighterhtml
const d = 'M150 0 L75 200 L225 200 Z'
render(document.body, () => svg`
  <svg height="210" width="400">
    <path .d=${d} />
  </svg>
`)
```

Please let me know if it doesn't, thanks!
